### PR TITLE
refactor: format code

### PR DESCRIPTION
**Pull request by wizard-28**
Refactors the shell code

### CODE REVIEW

File: misc/scripts/error_log.sh [hunk 1]
```diff
@@ -31,7 +31,7 @@ function error_log() {
 	local code="${1}"
 	local scope="${2}"
 
-	if [[ ! -f "$LOGFILE" ]]; then
+	if [[ ! -f $LOGFILE ]]; then
```
Comment (Henryws) [inline misc/scripts/error_log.sh:34]: Quote this.

File: misc/scripts/install-local.sh [hunk 2]
```diff
@@ -442,27 +444,33 @@ case "$url" in
...
 		if ! hashcheck "${url##*/}"; then
 			return 1
 		fi
 		# unzip file
-		unzip -q "${url##*/}" 1>&1 2>/dev/null
+		unzip -q "${url##*/}" 1>&1 2> /dev/null
 		# cd into it
-		cd ./*/ 2> /dev/null || ( error_log 1 "install $PACKAGE"; fancy_message warn "Could not enter into the downloaded archive" )
-	;;
+		cd ./*/ 2> /dev/null || (
```
Comment (Henryws) [inline misc/scripts/install-local.sh:463]: Use brackets.

Comment (Henryws) [inline misc/scripts/install-local.sh:463]: Use brackets.

File: misc/scripts/install-local.sh [hunk 3]
```diff
@@ -491,26 +501,29 @@ case "$url" in
...
 		download "$url"
 		# I think you get it by now
 		if ! hashcheck "${url##*/}"; then
 			return 1
 		fi
-		tar -xf "${url##*/}" 1>&1 2>/dev/null
-		cd ./*/ 2> /dev/null || ( error_log 1 "install $PACKAGE"; fancy_message warn "Could not enter into the downloaded archive" )
-	;;
+		tar -xf "${url##*/}" 1>&1 2> /dev/null
+		cd ./*/ 2> /dev/null || (
```
Comment (Henryws) [inline misc/scripts/install-local.sh:518]: Use brackets.

Comment (Henryws) [inline misc/scripts/install-local.sh:518]: Use brackets.

File: misc/scripts/install-local.sh [hunk 4]
```diff
@@ -567,19 +580,24 @@ if ! install; then
...
 
 if [[ $NOBUILDDEP -eq 1 ]]; then
-	sudo apt-get remove -y "$build_depends"
+	fancy_message info "Purging build dependencies"
+	# shellcheck disable=2086
+	sudo apt-get purge --auto-remove -y $build_depends
 fi
 
-cd "$HOME" 2> /dev/null || ( error_log 1 "install $PACKAGE"; fancy_message warn "Could not enter into ${HOME}" )
+cd "$HOME" 2> /dev/null || (
```
Comment (Henryws) [inline misc/scripts/install-local.sh:588]: Use brackets.

Comment (Henryws) [inline misc/scripts/install-local.sh:588]: Use brackets.

**Review by Henryws — CHANGES_REQUESTED** [review]
Requote everything.

File: misc/scripts/error_log.sh (hunk 1 above)
Comment (wizard-28) [reply misc/scripts/error_log.sh:34]: Okay the unquoting is due to the `-s` flag. You sure you want to get rid of the `-s` flag though? It has other benefits. Check the shfmt docs.

**Review by Henryws — APPROVED** [review]

**Comment by wizard-28** [comment]
@Henryws fix the merge conflict.

@D-Brox review.

**Review by D-Brox — APPROVED** [review]

**Comment by wizard-28** [comment]
Remaking due to merge conflicts.

**Pull request closed without merging**